packages/core/src/controllers/OnRampController.ts

```diff
@@ -104,8 +100,7 @@ export const OnRampController = {
 
   setSelectedProvider(provider: OnRampProvider | null) {
     if (provider && provider.name === 'meld') {
-      const pubKey =
-        process.env['NODE_ENV'] === 'production' ? MELD_PROD_PUBLIC_KEY : MELD_DEV_PUBLIC_KEY
+      const pubKey = process.env['NEXT_PUBLIC_MELD_DEV_KEY'] || ''
```

Comment:
Do we proceed with only one key? What happens to production key? Don't we replace it with new one?

Reply:
yes only pubkey is needed

Reply:
`process.env['NEXT_PUBLIC_MELD_DEV_KEY']` will work only in the lab